pkg/migration/migration.go

```diff
@@ -69,15 +77,11 @@ func (m *Migration) normalizeOptions() (stmts []*statement.AbstractStatement, er
 	if m.ReplicaMaxLag == 0 {
 		m.ReplicaMaxLag = 120 * time.Second
 	}
-	if m.Host == "" {
```

Comment:
this would have never been true as a normal course of operation since kong would inject a default at runtime